sctp/CHANGELOG.md

```diff
@@ -2,12 +2,14 @@
 
 ## Unreleased
 
+## 0.6.2
+
 * Increased minimum support rust version to `1.60.0`.
 * Do not loose data in `PollStream::poll_write` [#341](https://github.com/webrtc-rs/webrtc/pull/341).
 * `PollStream::poll_shutdown`: make sure to flush any writes before shutting down [#340](https://github.com/webrtc-rs/webrtc/pull/340)
 * Fixed a possible bug when adding chunks to pending queue [#345](https://github.com/webrtc-rs/webrtc/pull/345)
 
-## v0.6.1
```

Comment:
I think I've used the `v` prefix consistently elsewhere so let's keep it. I don't have strong feelings about whether we have it or not but I'd like to be consistent at least